shared/chat/conversation/attachment-fullscreen/container.tsx

```diff
@@ -76,24 +67,15 @@ const Connected = (props: OwnProps) => {
               currentDeviceName
             )
             if (goodMessage && goodMessage.type === 'attachment') {
-              setOrdinal(goodMessage.ordinal)
+              setMessage(goodMessage)
```

Comment:
this rpc gives us message we've never seen before sometimes so we need the message

Reply:
- [ ] is the menu broken?